js/views/playingScreen.js

```diff
@@ -229,6 +246,12 @@ var PlayingScreen = React.createClass({
           playerState={this.props.playerState}
           isLiveStream={this.props.isLiveStream} />
       </div>
+      {showUnmute ? <button className="oo-playing-screen oo-unmute"
+        onClick={this.unmuteClick}
+        aria-label={volumeAriaLabel}
+        >
+        <Icon {...this.props} icon={volumeIcon} ref="volumeIcon" />
+      </button> : null}
```

Comment:
Not needed currently, but this button would probably work better as a separate component once David's updates are added.

Reply:
Yep, agreed.